dnssec.go

```diff
@@ -450,9 +452,9 @@ func (rr *RRSIG) Verify(k *DNSKEY, rrset []RR) error {
 	switch rr.Algorithm {
 	case RSASHA1, RSASHA1NSEC3SHA1, RSASHA256, RSASHA512, RSAMD5:
 		// TODO(mg): this can be done quicker, ie. cache the pubkey data somewhere??
-		pubkey := k.publicKeyRSA() // Get the key
-		if pubkey == nil {
-			return ErrKey
+		pubkey, err := k.publicKeyRSA() // Get the key
+		if err != nil {
+			return err
```

Comment:
So ErrKey is public, and thus part of the API, so this change can't be made, because it backwards incompatible.
(didn't look this far in the code because of the unsafe import)